### PR TITLE
Remove prefix `enh_` and `diar_`

### PR DESCRIPTION
**Pull request by kan-bayashi**
Follow up of #3537 #3535 
- Removed the prefix `enh_` and `diar_`
- Cleaned `diar.sh`

### CODE REVIEW

**Comment by kan-bayashi** [comment]
@YushiUeda @Emrys365 Could you check it?

File: egs2/TEMPLATE/diar1/diar.sh [hunk 1]
```diff
@@ -485,8 +485,7 @@ if ! "${skip_eval}"; then
             _opts+="--config ${inference_config} "
         fi
 
-        # for dset in "${valid_set}" ${test_sets}; do
-        for dset in "${valid_set}"; do
+        for dset in "${valid_set}" ${test_sets}; do
```
Comment (kan-bayashi) [inline egs2/TEMPLATE/diar1/diar.sh:488]: This might be like this?

Comment (YushiUeda) [reply egs2/TEMPLATE/diar1/diar.sh:488]: Thanks for pointing out. I guess it's not intended, and should be as you modified.

**Comment by YushiUeda** [comment]
Thanks @kan-bayashi! I've checked the `diar_` files and they look good to me.

**Comment by Emrys365** [comment]
LGTM! Thanks a lot.

**Review by Emrys365 — APPROVED** [review]

**Pull request merged**